cmd/server/main.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/go-kratos/kratos/v2/transport/http"
 )
 
+//go:generate wire
```

Comment:
it already exists in wire_gen.go

Reply:
OK, I see.
I test it when wire_gen was removed, so I thought it was wrong.